envs/env.py

```diff
@@ -58,7 +59,8 @@ def __init__(self,
                  observation_length: int = len(State) * len(Statistic),
                  is_testing: bool = False,
                  max_duration: int = 80,
-                 max_time_steps_per_episode: int = 200):
+                 max_time_steps_per_episode: int = 200,
+                 **kwargs):
```

Comment:
This shouldn't be needed. [Env keyword arguments](https://rl-baselines3-zoo.readthedocs.io/en/master/guide/train.html#env-keyword-arguments) should be automatically matched.